conf/nutcracker.yml

```diff
@@ -65,3 +65,18 @@ omega:
   servers:
    - 127.0.0.1:11214:100000
    - 127.0.0.1:11215:1
+
+sigma:
+  listen: 127.0.0.1:22125
+  hash: fnv1a_64
+  distribution: ketama
+  auto_eject_hosts: false
+  redis: true
+  server_retry_timeout: 2000
+  server_failure_limit: 1
+  servers:
+    - 127.0.0.1:6379:1 server1
+    - 127.0.0.1:6380:1 server2
+  sentinels:
```

Comment:
To avoid being confused, I added one sentinel address to the example configuration.